### ISSUE TITLE
Programmatically close dialog(s)

### ISSUE THREAD

**Issue by kl3sk** [yariksav/vuetify-dialog#7]
Don't know if it's possible.

Thank again 👍

**Comment by yariksav** [comment]
You can put parameter waitForResult: false, then funciton will return dialog instance instead promise

``` js
      const dialogInstance = await this.$dialog[this.type]({
        title: this.title,
        text: this.text,
        waitForResult: false
      });
      /*
        `waitForResult` -  
        dialogInstance - is instance of vuedl component. 
        you can manualy close, or do some manual actions
        If you want to get user choise, just put
        const userChice = await dialogInstance.wait()
      */
      setTimeout(() => {
        dialogInstance.close()
      }, 3000)
      await dialogInstance.wait()

```

**Comment by yariksav** [comment]
See updated example

**Comment by kl3sk** [comment]
Thanks it work as well. Really good plugin 👍

**Comment by yariksav** [comment]
@kl3sk, i've fixed vuedl and in version v0.1.8 you can remove unnecessary condition "if (dialogInstance.vm) { ", just do
```js
      setTimeout(() => {
          dialogInstance.close()
      }, 3000)
```
